native/src/modules/categories/components/CategoryListContent.js

```diff
@@ -0,0 +1,179 @@
+// @flow
+
+import React from 'react'
+import { Linking, Dimensions, View, Text } from 'react-native'
+import Html, { GestureResponderEvent, type HTMLNode, type RendererFunction } from 'react-native-render-html'
+import { _constructStyles } from 'react-native-render-html/src/HTMLStyles'
+import styled from 'styled-components/native'
+import { type StyledComponent } from 'styled-components'
+import type { NavigationStackProp } from 'react-navigation-stack'
+import type { NavigateToInternalLinkParamsType } from '../../app/createNavigateToInternalLink'
+import buildConfig from '../../app/constants/buildConfig'
+import type { ThemeType } from '../../../../build-configs/ThemeType'
+import { RTL_LANGUAGES } from '../../i18n/constants'
+
+const VerticalPadding: StyledComponent<{}, {}, *> = styled.View`
+  padding: 0 20px;
+`
+type WindowEventType = {
+  window: {width: number,
+  height: number}
+}
+
+type ContentPropsType = {|
+  content: string,
+  navigation: NavigationStackProp<*>,
+  resourceCacheUrl: string,
+  navigateToInternalLink?: NavigateToInternalLinkParamsType => void,
+  cacheDictionary: {[string]: string},
```

Comment:
```suggestion
  cacheDictionary: { [string]: string },
```
Styling still not working for you?